src/actions/ast.js

```diff
@@ -24,11 +24,11 @@ export function setSymbols(sourceId: SourceId) {
     }
 
     const source = sourceRecord.toJS();
-    if (!source.text || hasSymbols(getState(), source)) {
+    if (!source.text) {
       return;
     }
 
-    const symbols = await parser.getSymbols(source);
+    const symbols = await parser.getSymbols(source, true);
```

Comment:
if we're always updating here do we need the boolean?

Reply:
we have other cases where we use getSymbols